okhttp/src/main/java/okhttp3/android/android.kt

```diff
@@ -0,0 +1,53 @@
+package okhttp3.android
+
+import android.content.Context
+import okhttp3.Cache
+import okhttp3.OkHttpClient
+import okhttp3.internal.platform.Platform
+import okhttp3.internal.platform.android.BuildConfig
+import okhttp3.internal.tls.AllowlistedTrustManager
+
+const val MiB = 1024L * 1024L
+
+fun setDevMode(buildConfigClass: Class<*>? = null, buildConfig: BuildConfig? = null) {
```

Comment:
the `static` nature of this feels bad. . . who is responsible for calling this?

If we wanted to do this, I think we should ship an aar and install it automatically? (Should we do that?)

Reply:
Let's try it.  Don't rush this. Maybe it's not due for 5.0 branch along with other eg MP features. But get it feeling right before we make a bad public API.

Reply:
I think that also solves another issue I had.  Outside of robolectric android-all anything else is mostly unavailable if shipped as an AAR.  This code could live in a AAR building okhttp-android module?

Reply:
Yep!